scapy/sendrecv.py

```diff
@@ -140,6 +140,8 @@ def sndrcv(pks, pkt, timeout = None, inter = 0, verbose=None, chainCC=0, retry=0
                                 inp = []
                                 try:
                                     inp, out, err = select(inmask,[],[], remaintime)
+                                except error as e:
```

Comment:
I don't really like continuing like that, as select.error can be differents errors...

As the `select.error` is something like
`(4, 'Interrupted system call')`
Couldn't we check if the error is 4, and if so continue, otherwise raise ?